mono/metadata/object.c

```diff
@@ -861,6 +861,8 @@ compute_class_bitmap (MonoClass *klass, gsize *bitmap, int size, int offset, int
 
 	wordsize = TARGET_SIZEOF_VOID_P;
 
+	error_init (error);
```

Comment:
This is not really needed, the caller needs to initialize it.

Reply:
Ah okay. It looked to me like there was a pattern where any function that accepted a `MonoError*` would call `error_init` on it to init/reset the error object. If this is not the case I can remove the redundant inits. :-)